lib/features/composer/presentation/extensions/setup_email_content_extension.dart

```diff
@@ -130,7 +66,10 @@ extension SetupEmailContentExtension on ComposerController {
       case EmailActionType.replyAll:
       case EmailActionType.forward:
         if (arguments.emailContents?.trim().isNotEmpty != true) {
-          emailContentsViewState.value = Left(GetEmailContentFailure(EmptyEmailContentException()));
+          await _getEmailContent(
+            arguments,
+            transformConfiguration: TransformConfiguration.forReplyForwardEmail(),
```

Comment:
Should be create a new transformation for `Reply & Forward` with empty content. Avoid having to call `ImageTransformers` again when the content has already been processed on the email view